contents/docs/experiments/common-questions.mdx

```diff
@@ -1,46 +1,42 @@
+2. Create a new experiment and give your feature flag the same key as the feature flag you deleted in step 1.
+3. Name the first variant in your new feature flag 'control'.
 
-### How do I run a second experiment using the same feature flag as the first experiment? I don't want to delete experiment #1.
+![Reuse an existing feature flag for an experiment](../../images/features/experiments/reuse-feature-flag-for-experiment.png)
 
-The same caveats as above apply. If you want to re-run an experiment (using the same feature flag key) while preserving the previous experiment results, simply delete the existing feature flag (not the experiment) and use the same key in the new experiment.
+**Note:** Deleting the flag is equivalent to it being disabled, so it will be off for however long it takes you to create the draft experiment. The flag will be enabled as soon as the experiment is created (not launched).
 
----
+## How do I run a second experiment using the same feature flag as the first experiment?
 
+This is similar to running an experiment using an [exsting feature flag](/docs/experiments/common-questions#how-do-i-use-an-existing-feature-flag-in-an-experiment). If you want to re-run an experiment (using the same feature flag key) while preserving the previous experiment results, delete the existing feature flag (not the experiment) and use the same key in the new experiment.
 
 ## I want to run experiments in PostHog with my custom feature flag setup, how can I do this?
 
-Broadly, a few things to be aware of:
+See our docs on [how to run an experiment without using PostHog feature flags](http://localhost:8001/docs/experiments/running-experiments-without-feature-flags).
 
-1. Events need to be enriched with feature flag information for them to work in experiments.
-2. If you're running experiments with a trend goal, you also need to send a `$feature_flag_called` event.
-3. For funnel experiments, having at least one event with flag information is sufficient. So, if you have backend events in your experiment, you don't have to wrangle to figure out how to add flag information to them, you can just use a frontend event that automatically gets this information.
+## How do I assign a specific person to the control/test variant in an experiment?
 
-If you are not sure if you've handled the above three things, [read an in-depth guide here](/docs/experiments/running-experiments-without-feature-flags)
+Once you create the experiment, go to the feature flag, scroll down to "Release Conditions". For each condition you'll see an "Optional Override". This allows you to choose a release condition and force all people in this release condition to have the variant chosen in the optional override.
 
----
-
-## How do I assign a specific person to control/test variant in an experiment?
+## My "Feature Flag Called" events show responses of "none" or "false" in addition "control" and "test"
```

Comment:
Referring to the "Feature Flag Response" property? The event itself doesn't show anything, it's the property.